app/experimenter/experiments/models/nimbus.py

```diff
@@ -30,6 +30,7 @@ class NimbusExperiment(NimbusConstants, models.Model):
     public_description = models.TextField(default="")
     risk_mitigation_link = models.URLField(max_length=255, blank=True)
     is_paused = models.BooleanField(default=False)
+    is_ended = models.BooleanField(default=False)
```

Comment:
I think we want this to be the other way around.  When an experiment is complete it moves to `status=COMPLETE`, so I don't think we need a bool to track when it's actually ended, we want something to track when it's REQUESTED to be ended, something like `is_end_requested`.